persistence/src/main/java/fi/otavanopisto/pyramus/dao/PyramusEntityDAO.java

```diff
@@ -99,7 +99,12 @@ protected void addTokenizedSearchCriteria(StringBuilder queryBuilder, String fie
   
   protected void addTokenizedSearchCriteria(StringBuilder queryBuilder, boolean required, String value, String ... fields) {
     String inputText = value.replaceAll(" +", " ");
-    String[] tokens = escapeSearchCriteria(inputText).split("[ ,]");
+    String[] tokens = new String[] {inputText};
+    
+    if (!fields[0].contains("Emails")) {
```

Comment:
Should we just make a copy of this whole method as `addUntokenizedSearchCriteria` that would just skip the `escapeSearchCriteria` always and return this method to its' old state? I'm not a big fan of relying on hard-coded values that alter the behaviour on the fly.